README.md

```diff
@@ -366,6 +367,18 @@ To use an easy to reproduce example, see [minikube.jsonnet](examples/minikube.js
 
 Working examples of use with continuous delivery tools are found in examples/continuous-delivery.
 
+## Security Scanning
```

Comment:
Could we change the name of this section to `Security` and move the content to a separate file (to keep https://github.com/prometheus-operator/kube-prometheus/issues/1005 in mind).

This way we can have a quick reference for all security-related aspects in one place.